build.sbt

```diff
@@ -544,7 +544,7 @@ lazy val avroFlinkUtil = (project in engine("flink/avro-util")).
       )
     }
   )
-  .dependsOn(kafkaFlinkUtil, kafkaTestUtil % "test", flinkTestUtil % "test", process % "test")
```

Comment:
maybe we can have SerializersRegistrar in flink-api?

Reply:
fixed